io_scene_nif/nif_import.py

```diff
@@ -216,9 +214,9 @@ def execute(self):
 
     def import_root(self, root_block):
         """Main import function."""
-        # check that this is not a kf file
-        if isinstance(root_block, (NifFormat.NiSequence, NifFormat.NiSequenceStreamHelper)):
-            raise nif_utils.NifError("direct .kf import not supported")
+        # # check that this is not a kf file
```

Comment:
Is this still not needed in the case of this type.

Reply:
Can probably removed or adjusted to run the load_kf_standalone function?

Reply:
Changed the error to read: "Use the KF import operator to load KF files."
I think it's best to keep two different operators for KF and NIF and avoid muddling the border between them by allowing the NIF one to still read KF files.
And a one-for-all NIF and KF operator is also probably counterintuitive and less transparent. If the file selector's file extension mask showed both KF & NIF files it could be confusing.